### PR TITLE
Cleaning up static addressing info and clarifying details.

### PR DESCRIPTION
**Pull request by marshallmassengill**
I have this funny feeling that I'm opening a can of worms with this proposed change but this should clarify a few things.  Hopefully this helps.  If you don't use it, that's totally ok too. The only thing I'm not really happy about is the use of "field router" but that is what the device is called.  A more generic "Field Networking Equipment" could be substituted.

### CODE REVIEW

File: source/docs/networking/networking-introduction/ip-configurations.rst [hunk 1]
```diff
@@ -26,7 +26,7 @@ This section describes networking when connected to the Field Network for match
 On the Field DHCP Configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Field Network runs a DHCP server with pools for each team that will hand our addresses in the range of 10.TE.AM.20 and up with subnet masks
+The Field Network runs a DHCP server with pools for each team that will hand out addresses in the range of 10.TE.AM.20 to 10.TE.AM.199 with a subnet mask of 255.255.255.0, and a default gateway of 10.TE.AM.4.
```
Comment (AustinShalit) [inline source/docs/networking/networking-introduction/ip-configurations.rst:29]: While we are here we probably should clarify that the radio has its own DHCP server that passes out IPs to the robot local network. And the field only passes IPs to the local field network. No DHCP traffic should flow over the wireless link.

File: source/docs/networking/networking-introduction/ip-configurations.rst [hunk 2]
```diff
@@ -39,13 +39,13 @@ The Field Network runs a DHCP server with pools for each team that will hand our
...
+It is also possible to configure static IPs on your devices to accommodate devices or software which do not support mDNS. When doing so you want to make sure to avoid addresses that will be in use when the robot is on the field network. These addresses are ``10.TE.AM.1`` for the OpenMesh radio, ``10.TE.AM.4`` for the field router, and anything greater than ``10.TE.AM.20`` which may be assigned to a device configured for DHCP or else reserved. The roboRIO network configuration can be set from the webdashboard.
 
 -  OpenMesh radio - Static ``10.TE.AM.1`` programmed by Kiosk
 -  roboRIO - Static ``10.TE.AM.2`` would be a reasonable choice, subnet mask
    of ``255.255.255.0`` (default)
 -  Driver Station - Static ``10.TE.AM.5`` would be a reasonable choice,
-   subnet mask **must** be ``255.0.0.0``
+   subnet mask **must** be ``255.0.0.0`` to enable the DS to reach both the robot and FMS Server
```
Comment (AustinShalit) [inline source/docs/networking/networking-introduction/ip-configurations.rst:48]: Let's add a "without additional configuration" so folks don't get confused why DHCP works.

**Review by AustinShalit — CHANGES_REQUESTED** [review]

File: source/docs/networking/networking-introduction/ip-configurations.rst (hunk 2 above)
Comment (marshallmassengill) [reply source/docs/networking/networking-introduction/ip-configurations.rst:48]: Agreed and if we're going to do this then I'd suggest being up front about the alternative, which is to configure the default gateway.

File: source/docs/networking/networking-introduction/ip-configurations.rst (hunk 1 above)
Comment (marshallmassengill) [reply source/docs/networking/networking-introduction/ip-configurations.rst:29]: Agreed.  That would help add additional context.

File: source/docs/networking/networking-introduction/ip-configurations.rst (hunk 2 above)
Comment (AustinShalit) [reply source/docs/networking/networking-introduction/ip-configurations.rst:48]: Works for me

**Review by AustinShalit — APPROVED** [review]

**Review by Daltz333 — DISMISSED** [review]

**Review by Daltz333 — APPROVED** [review]

File: source/docs/networking/networking-introduction/ip-configurations.rst [hunk 3]
```diff
@@ -26,7 +26,8 @@ This section describes networking when connected to the Field Network for match
 On the Field DHCP Configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The Field Network runs a DHCP server with pools for each team that will hand our addresses in the range of 10.TE.AM.20 and up with subnet masks
+The Field Network runs a DHCP server with pools for each team that will hand out addresses in the range of 10.TE.AM.20 to 10.TE.AM.199 with a subnet mask of 255.255.255.0, and a default gateway of 10.TE.AM.4.
```
Comment (Daltz333) [inline source/docs/networking/networking-introduction/ip-configurations.rst:29]: While we are here 
```suggestion
The Field Network runs a DHCP server with pools for each team that will hand out addresses in the range of ``10.TE.AM.20`` to ``10.TE.AM.199`` with a subnet mask of ``255.255.255.0``, and a default gateway of ``10.TE.AM.4``.
```

**Review by Daltz333 — CHANGES_REQUESTED** [review]

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**